src/ios/OpenTokPlugin.m

```diff
@@ -610,6 +610,10 @@ - (void)sessionDidDisconnect:(OTSession*)session{
     if( _publisher ){
         [_publisher.view removeFromSuperview];
     }
+    // Remove session observers
+    for ( id key in streamDictionary ) {
+        [self removeObserversFromStream: [streamDictionary objectForKey:key]];
+    }
```

Comment:
We should maybe also remove all objects within the streamDictionary then for memory sake(like a few lines above) `[streamDictionary removeAllObjects]`.

Reply:
I agree that we should free up resources and remove all objects. @amatakasap can you please update the PR?